src/runner.go

```diff
@@ -96,26 +105,29 @@ func processSignedScript(incomingContent []byte) string {
 	}
 	log.Infoln("Signature of yaml file is valid")
 
-	// Parse the YAML data into the yamlConfig struct
-	var yamlContent signedYamlFile
+	// Parse the YAML data into array consisting of items of expected structure
+	var yamlContent []signedYamlFile
```

Comment:
What about calling this `yamlContentArray`? or `signedYamlContent` 

This way we can change the `yamlContentFirstItem` to just `yamlContent` making no distinction about other positions in this array, meaning that the position 0 is the "official" content for us.

Reply:
Went with `signedYamlArray` , also renamed the structs to match the expected content: `signedYamlContent` and `signedYamlContentVars` , also both have docstrings  now.

`yamlContentFirstItem` is now just `yamlContent` as suggested